contracts/core/extensions/CoreInternal.sol

```diff
@@ -104,6 +104,47 @@ contract CoreInternal is
         state.setTokens = state.setTokens.remove(_set);
     }
 
+    /**
+     * Add a price library to the mapping of tracked price libraries. Can only be set by
+     * owner of Core.
+     *
+     * @param  _priceLibrary   The address of the Price Library to enable
+     */
+    function enablePriceLibrary(
+        address _priceLibrary
+    )
+        external
+        onlyOwner
+    {
+        // Mark as true in validPriceLibraries mapping
+        state.validPriceLibraries[_priceLibrary] = true;
+
+        // Add to priceLibraries array
+        state.priceLibraries.push(_priceLibrary);
+    }
+
+    /**
+     * Disable a priceLibrary in the mapping of tracked Price Libraries. Can only be disabled
+     * by owner of Core.
+     *
+     * @param  _priceLibrary   The address of the Price Library to disable
+     */
+    function disablePriceLibrary(
```

Comment:
Why do we also need a disable method? Can we instead have one setter method where we set whether a price library is enabled?

Reply:
We could. I was just following the pattern we already have in place.

Reply:
Yeah it doesn't make sense to have two methods, lets add a card to get the other ones changed as well (also making `disableSet` bi directional). I'm also not sure we need the array and the map.

Reply:
https://trello.com/c/3pNq8JyQ/640-refactor-coreinternal-add-remove-functions-into-a-single-setter-function